corehq/apps/data_analytics/tasks.py

```diff
@@ -0,0 +1,36 @@
+from celery.schedules import crontab
+from celery.task import periodic_task
+from celery.utils.log import get_task_logger
+import datetime
+
+from corehq.apps.data_analytics.malt_generator import MALTTableGenerator
+from dimagi.utils.dates import DateSpan
+from dimagi.utils.django.email import send_HTML_email
+
+
+logger = get_task_logger(__name__)
```

Comment:
The malt-generation command logs errors. Will those logs be directed to the celery logger as well. I have tried to figure out if the error-logs will be sent as email, but I couldn't figure out.